### PR TITLE
Make a nicer menu

### PR DESCRIPTION
**Pull request by u9g**

### CODE REVIEW

**Comment by u9g** [comment]
closes https://github.com/PrismarineJS/prismarine-web-client/issues/76

**Comment by u9g** [comment]
this pr splits up the host:port into two seperate variables

**Comment by u9g** [comment]
Here's a video of what the login prompt looks like https://streamable.com/a8q8ig

File: lib/menu.js [hunk 1]
```diff
@@ -6,51 +6,238 @@ class PrismarineMenu extends LitElement {
...
     }
   }
 
   static get styles () {
     return css`
-        label {
-            grid-column: labels;
-            grid-row: auto;
-        }
+    html {
```
Comment (rom1504) [inline lib/menu.js:29]: html is outside of the web component, you cannot affect it
You can use :host instead if you want to affect the host element

File: lib/menu.js [hunk 2]
```diff
@@ -6,51 +6,238 @@ class PrismarineMenu extends LitElement {
...
+      height: 100%;
+    }
+    body {
+      margin:0;
+      padding:0;
+      font-family: sans-serif;
```
Comment (rom1504) [inline lib/menu.js:35]: body is outside you cannot affect it
You can use * to apply the font to all elements of the web component

File: styles.css [hunk 3]
```diff
@@ -9,14 +9,14 @@
...
 }
-
-html,
 body {
-	height: 100%;
-	margin: 0;
-	padding: 0;
+  margin:0;
+  padding:0;
+  font-family: sans-serif;
```
Comment (rom1504) [inline styles.css:18]: If you want to affect only the font of the web component you need to put that inside it

**Review by rom1504 — CHANGES_REQUESTED** [review]
Nice

I put some technical comments

**Comment by Karang** [comment]
nice

**Review by rom1504 — APPROVED** [review]
checked it, it looks good, those are details, merging

**Pull request merged**